src/components/composer/grid_composer/grid_composer.ts

```diff
@@ -165,12 +170,10 @@ export class GridComposer extends Component<Props, SpreadsheetChildEnv> {
 
       background,
       color,
-
-      "font-size": `${fontSizeMap[fontSize]}px`,
-      "font-weight": String(fontWeight),
+      "font-size": `${fontSizeInPixels(fontSize)}px`,
+      "font-weight": fontWeight === "" ? undefined : String(fontWeight),
```

Comment:
```suggestion
      "font-weight": fontWeight,
```

Reply:
With my suggestion above you can do this. Anyway, it's useless to cast fontWeight as a String, as it's already a string